### PR TITLE
Updating ActiveState publishing docs

### PR DESCRIPTION
**Pull request by rawktron**
A few small corrections and some updates to ActiveState Trusted Publisher docs.

### CODE REVIEW

**Comment by rawktron** [comment]
@di Hey Dustin, just made a few tweaks to the docs here. Text changes only. Hoping to merge before we go live. Thanks!

File: docs/user/trusted-publishers/using-a-publisher.md [hunk 1]
```diff
@@ -270,44 +270,53 @@ below describe the setup process for each supported trusted publisher.
 
 === "ActiveState"
 
-    ActiveState's Platform works as a zero-config CI solution for your dependencies to automatically build cross-platform wheels of your PyPI projects. Once you're set up on the Platform and have linked your PyPI project, you're ready to publish. For more information on getting started with ActiveState, go [here](https://docs.activestate.com/platform/start/PYPI). To begin:
+    ActiveState's Platform works as a zero-config CI solution for your dependencies to automatically build cross-platform wheels of your PyPI projects. Once you're set up on the Platform and have linked your PyPI project, you're ready to publish. For more information on getting started with ActiveState, go [here](https://docs.activestate.com/platform/start/pypi). To begin:
```
[the remark this answers is not in the document]
Comment (di) [reply docs/user/trusted-publishers/using-a-publisher.md:273]: You can see the preview here: https://docspypiorg--15739.org.readthedocs.build/trusted-publishers/using-a-publisher/#activestate

[the remark this answers is not in the document]
Comment (rawktron) [reply docs/user/trusted-publishers/using-a-publisher.md:273]: Oh yeah good catch. Looks good to me @di !

[the remark this answers is not in the document]
Comment (di) [reply docs/user/trusted-publishers/using-a-publisher.md:273]: You'll have to accept the suggestion or push a commit, I can't make changes to your branch :)

**Review by di — APPROVED** [review]

**Pull request merged**